modules/ROOT/pages/aura-cli/installation.adoc

```diff
@@ -0,0 +1,44 @@
+= Installation
+:description: Install the Neo4j Aura command line interface.
+
+The Aura CLI is available on link:https://github.com/neo4j/aura-cli/releases[GitHub].
```

Comment:
Add information about  Aura so people know what version of Aura the CLi works with.

Suggested to add at the end of the sentence 

_and works with all versions of the Aura API_